src/Settings.js

```diff
@@ -137,10 +158,11 @@ export class Settings {
      *      featureToggles: AutofillFeatureToggles,
      *      enabled: boolean | null
      * }>}
-     * @param {AvailableInputTypes} [availableInputTypesOverrides] a migration aid so that macOS can provide data in its old way initially
      */
-    async refresh (availableInputTypesOverrides) {
+    async refresh () {
```

Comment:
Thanks for incrementally improving this as part of your PR. Less params/mental load, and this is looking so much cleaner now 💪